cmd/crd-puller/pull-crds.go

```diff
@@ -54,7 +54,7 @@ func main() {
 			if err != nil {
 				return err
 			}
-			crds, err := puller.PullCRDs(context.TODO(), args...)
+			crds, err := puller.PullCRDs(context.TODO(), resourcesToSync...)
```

Comment:
wasn't args doing the same, just without a flag in front of the list?

Reply:
Yes. I thought it would be nice to provide a flag and standardize it similar to the --kubeconfig flag being used right now. Also, if someone is using a script to call pull-crds.go, its nicer to use flags instead of args :)

Reply:
do we actually want anybody to call crd-puller? Isn't it going away when we have fixed the pull mode syncer?

Reply:
Ok. I didn't think about pull mode syncer being the one used moving forward.
If this is not really useful, I can close the PR.